### PR TITLE
Adding Watson TTS (IBM Cloud)

### PR DESCRIPTION
**Pull request by rutkai**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant):** home-assistant/home-assistant#23299

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/watson_tts.markdown [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+description: "Instructions on how to setup IBM Watson TTS with Home Assistant."
+date: 2019-04-22 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: watson_tts.png
+ha_category: Text-to-speech
+ha_release: 0.92
+redirect_from:
```
Comment (klaasnicolaas) [inline source/_components/watson_tts.markdown]: Not necessary because this is not an existing page.

File: source/_components/watson_tts.markdown [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+title: "Watson TTS"
+description: "Instructions on how to setup IBM Watson TTS with Home Assistant."
+date: 2019-04-22 12:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: watson_tts.png
+ha_category: Text-to-speech
+ha_release: 0.92
```
Comment (klaasnicolaas) [inline source/_components/watson_tts.markdown]:
```suggestion
ha_release: 0.93
```

File: source/_components/watson_tts.markdown [hunk 3]
```diff
@@ -0,0 +1,110 @@
+---
```
Comment (klaasnicolaas) [inline source/_components/watson_tts.markdown:1]: Pherhaps also add the IOT class

File: source/_components/watson_tts.markdown [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+## {% linkable_title Configuration %}
+
+To configure Watson TTS, add the following lines to your `configuration.yaml`:
+
+```yaml
+# Example configuration.yaml entry
+tts:
+  - platform: watson_tts
+    watson_url: https://stream.watsonplatform.net/text-to-speech/api/v1/synthesize
```
Comment (klaasnicolaas) [inline source/_components/watson_tts.markdown]: Only config variables that are required in the first example

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/watson_tts.markdown (hunk 3 above)
Comment (rutkai) [reply source/_components/watson_tts.markdown:1]: Which class should it be? I checked the other TTS services and no other one has this field.

Comment (rutkai) [reply source/_components/watson_tts.markdown:1]: @klaasnicolaas :point_up: ?

Comment (klaasnicolaas) [reply source/_components/watson_tts.markdown:1]: If we don't know then we can leave it out.

Comment (rutkai) [reply source/_components/watson_tts.markdown:1]: Okay then

**Review by klaasnicolaas — APPROVED** [review]

**Pull request merged**